example/test_problem/ELBDM/PlaneWave/Input__TestProb

```diff
@@ -0,0 +1,6 @@
+# problem-specific runtime parameters
+PWave_Lambda     0.5           # plane wave wavelength [0.5]
```

Comment:
Don't need this if you decide to use periodic BC.

Reply:
Done.